packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -182,9 +203,31 @@ export default class Sayt extends LitElement {
       this.hideSayt();
       return;
     }
+    this.debouncedRequestSaytAutocompleteTerms(query, searchbox);
+    this.debouncedRequestSaytProducts(query, searchbox);
```

Comment:
The `handleAutocompleteTermHover` function should also be using `debouncedRequestSaytProducts`.

Reply:
It does not below on line 266 if that is correct